src/components/TodoInfo/TodoInfo.scss

```diff
@@ -0,0 +1,18 @@
+.TodoInfo {
+  width: max-content;
+  padding: 8px;
+  margin: 12px 0;
+  border: 1px solid #000;
+  border-radius: 8px;
+  background-color: antiquewhite;
```

Comment:
Consider using a more appropriate background color for better user experience. 🎨